README.md

```diff
@@ -17,6 +17,11 @@ Python packages with our recommended tooling set up and ready to go.
 > package template (e.g., [`scikit-hep`](https://github.com/scikit-hep/cookie)),
 > we recommend using their template instead of this one.
 
+## Scope and contributing
+
+This project is written by members of ARC for members of ARC, and is scoped to be a helpful tool for the projects we work on in ARC.
+As such contributions to this repository are only open to members ARC, but we welcome (and encourage!) suggestions for improvements or bug fixes from anyone.
```

Comment:
There already is a CONTRIBUTING.md with

> This template and our [recommendation pages](https://github-pages.arc.ucl.ac.uk/python-tooling) were made by [research software engineers](https://society-rse.org/about/history) at [UCL's Centre for Advanced Research Computing](https://ucl.ac.uk/arc). We made it with research software projects in mind, but whoever you are, we hope you find this useful!

Does that make the scope clear enough?